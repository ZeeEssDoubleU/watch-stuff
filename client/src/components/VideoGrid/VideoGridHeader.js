import React from "react";
import PropTypes from "prop-types";

import "./VideogridHeader.scss";

const VideoGridHeader = props => {
	return (
		<div className="video-grid-header">
			<span className="title">{props.title}</span>
		</div>
	);
};

VideoGridHeader.propTypes = {};

export default VideoGridHeader;
