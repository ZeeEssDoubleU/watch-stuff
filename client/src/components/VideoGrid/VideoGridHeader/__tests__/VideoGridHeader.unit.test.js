import React from "react";
import { shallow } from "enzyme";

import VideoGridHeader from "../VideoGridHeader";

describe("VideoGridHeader", () => {
	test("renders without props", () => {
		const wrapper = shallow(<VideoGridHeader />);
		expect(wrapper).toMatchSnapshot();
	});

	test("renders title prop empty", () => {
		const wrapper = shallow(<VideoGridHeader title="" />);
		expect(wrapper).toMatchSnapshot();
	});

	test("renders with title prop", () => {
		const wrapper = shallow(<VideoGridHeader title={"Trending"} />);
		expect(wrapper).toMatchSnapshot();
	});
});
