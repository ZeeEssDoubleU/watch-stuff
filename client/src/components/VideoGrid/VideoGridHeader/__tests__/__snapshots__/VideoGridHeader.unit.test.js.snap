// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeader renders title prop empty 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`VideoGridHeader renders with title prop 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    Trending
  </span>
</div>
`;

exports[`VideoGridHeader renders without props 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;
